x-pack/legacy/plugins/maps/public/register_vis_type_alias.js

```diff
@@ -37,5 +37,5 @@ visualizationsSetup.types.registerAlias({
 });
 
 if (!showMapVisualizationTypes) {
-  visualizationsSetup.types.hideTypes(['region_map', 'tile_map']);
+  //visualizationsSetup.types.hideTypes(['region_map', 'tile_map']);
```

Comment:
this line should remain so region_map and tile_map are hidden in the create new visualization menu.

Reply:
yup, i uncommented this to help me test